.github/workflows/publish-admin-ui-components.yml

```diff
@@ -30,19 +34,24 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
       - name: Install dependencies
         run: yarn install
```

Comment:
Here I think we could use `yarn install --production` since at this point we don't need any dev dependencies. (I just noticed this same thing for `ui-components`)

Reply:
done!